awl/mslider.cpp

```diff
@@ -234,15 +234,22 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
    	p.setFont(f);
       p.setPen(QPen(Qt::darkGray, 2));
    	QFontMetrics fm(f);
-      int xt = 20 - fm.width("00") - 5;
+      int xt = 20 - fm.width("-00") - 5;
 
       QString s;
-   	for (int i = 10; i < 70; i += 10) {
-      	h  = y1 + lrint(i * mh / range);
-         	s.setNum(i - 10);
-  		p.drawText(xt,  h - 3, s);
-		p.drawLine(15, h, 20, h);
-         	}
+      for (int i = minValue(); i <= maxValue(); i += 10) {
+            h  = y1 + lrint((maxValue() - i) * mh / range);
+            s.setNum(i);
+            if (i > 0)
+                  p.drawText(QRect(xt - 10, h - 4, 20, 8), Qt::AlignRight, QString::fromStdString("+") + s);
+            else if (i == minValue()) {
+                  p.drawText(QRect(xt - 10, h - 4, 20, 8), Qt::AlignRight, QString::fromStdString("-∞"));
```

Comment:
@Jojo-Schmitz upon further study, it seems that u8 prefix is needed since QString::fromStdString() [specs say](http://doc.qt.io/qt-5/qstring.html#fromStdString) "The given string is converted to Unicode using the fromUtf8() function."